frontend/models/product-list/server.ts

```diff
@@ -42,15 +43,19 @@ export async function findProductList(
    deviceItemType: string | null = null
 ): Promise<ProductList | null> {
    const filterDeviceTitle = filters.deviceTitle?.eqi ?? '';
+   const isPartsList = filterDeviceTitle || filters.handle?.eq === 'Parts';
```

Comment:
I suggest to use `productListType` enum to assert  this, instead of hard-coding the url

Reply:
Good idea https://github.com/iFixit/react-commerce/pull/1252/commits/2d8be9df791fd5789d6ff4b43ce63786858755b5

In order to make the requests in parallel, I chose to pass the product list type into this function instead of waiting for the product list to resolve to get the type.

This commit uses the product list type from the caller in the product list object, whereas before we used the product list type from Strapi. If you think this will cause problems, let me know, but it seemed unlikely that the two would be different unless the Strapi data was invalid.